index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <title>SA AUNG HTET NYEIN</title>
+  </head>
+  <body>
+    <!-- the toolbar -->
+    <nav>
+      <p class="logo">SASA</p>
+      <div class="menu"><i class="fa-solid fa-bars"></i></div>
+    </nav>
+
+    <!-- headline -->
+    <section id="headline">
+      <h1 class="big-intro">Hey There. I’m SASA</h1>
+      <h5 class="small-intro">I’am a Software Developer</h5>
+      <p class="about-me">
+        A software engineering student with holistic knowledge of software
+        development with over 4 years of web development experience. Passionate
+        about UI/UX design and the art of photography. Extremely motivated to
+        develop my skills and grow professionally.
+      </p>
+      <div class="social-accounts">
+        <a href="#">
+          <i class="fa-brands fa-github"></i>
+        </a>
+        <a href="#">
+          <i class="fa-brands fa-linkedin"></i>
+        </a>
+        <a href="#">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="#">
+          <i class="fa-brands fa-medium"></i>
+        </a>
+      </div>
+      <img
+        class="illustration"
+        src="./assets/images/headline-section-illustration.png"
+        alt=""
+      />
```

Comment:
- [ ] Kindly make sure this is a `background-image` not an `img` tag. 🙏  If an image isn't part of your website content but for aesthetic reasons then a `background-image` is preferred. You can read more about it [here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)